syncer/sharding_group.go

```diff
@@ -295,6 +303,8 @@ func (sg *ShardingGroup) UnresolvedTables() [][]string {
 	sg.RLock()
 	defer sg.RUnlock()
 
+	// TODO: if we have sharding ddl sequence, and partial ddls synced, we treat
```

Comment:
Not accurate enough

Reply:
Then, we can't forward the checkpoint of tables?